addons/website_slides/__manifest__.py

```diff
@@ -30,6 +30,7 @@
         'views/slide_slide_views.xml',
         'views/slide_channel_views.xml',
         'views/website_slides_menu_views.xml',
+        'views/website_slides_profile.xml',
```

Comment:
Guidelines: website_slides_templates_profile.xml . a bit verbose but that way we know those are templates, linked to the profile application/ feature :) . To check in other commits to have a coherent naming.